sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 from typing import Any, Union, Optional, TYPE_CHECKING
+from typing_extensions import Literal
```

Comment:
I planned to do this:
```
try:
  from typing import Literal
except ImportError:
  from typing_extensions import Literal
```
but `typing.Literal` does not exist in Python 3.7, so it results in a mypy error. I'm not sure if there's a better way to do this? @annatisch